### ISSUE TITLE
feat(les_fvm): FV transport emulator with emulator-adjoint calibration harness (Step 3)

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/plumax#95]
## Problem / Request
Tier III's Step 3 (`les_fvm.emulator`, `les_fvm.emulator_adjoint_test`) is ☐. The design's "hard test" — emulator-autodiff gradients vs FV-autodiff gradients within 5 % in operator norm on held-out met — is what makes Step 4 inversion trustworthy; failure means a biased posterior. The FV solver's own adjoint memory cost (long windows need checkpointing) is the same question the emulator's multi-step rollout training faces, so this issue also benchmarks the checkpointing strategy the design leaves open.

## User Story
> As a user running many-day windows, I want a neural surrogate for the FV time-stepper whose gradients are validated against the exact discrete adjoint, so 4D-Var can run on the surrogate without silently biasing the source posterior.

## Proposed API
```python
# les_fvm/emulator.py
class TransportEmulator(eqx.Module):     # UNet v1; FNO/GNN as later options
    def step(self, c: Float[Array, "Nz Ny Nx"], wind_t, source_t, dt) -> Float[Array, "Nz Ny Nx"]: ...
    def rollout(self, c0, winds, sources, dt, n_steps) -> Float[Array, "T Nz Ny Nx"]: ...

def generate_rollouts(key, *, grid, met_sampler, n_cases, n_steps, dt) -> tuple[...]: ...   # via simulate_eulerian_dispersion
def train_transport_emulator(model, data, *, steps, lr, horizon_ramp: bool = True) -> TransportEmulator: ...
def emulated_forward(emu, problem: FourDVarProblem) -> EulerianForward4DVar: ...   # drop-in for build_forward

# les_fvm/emulator_adjoint_test.py
def gradient_operator_norm_error(emu, problem, *, key, n_probes: int = 16) -> float: ...
```

## Implementation Steps
- [ ] Add `les_fvm/emulator.py` (equinox UNet, mass-conserving output layer)
- [ ] Multi-step rollout training with ramped horizon (design: "multi-step rollout with ramped horizon" to fight drift)
- [ ] `emulated_forward` builds an `EulerianForward4DVar` whose `predict` uses `emu.rollout`, so `solve_4dvar` is unchanged
- [ ] Calibration harness comparing `jax.vjp` through the emulator and through `simulate_eulerian_dispersion` on random probes
- [ ] Benchmark `equinox.internal.scan_checkpointed` vs plain `scan` memory/time for the FV adjoint at 3 window lengths; record in the docs

## Definition of Done
- [ ] Gradient operator-norm error < 5 % on held-out met
- [ ] OOD check: trained on one met regime, error on another reported (no threshold, but recorded)
- [ ] `make test && make lint && make typecheck` green (`slow` for training)

## Testing
- [ ] `tests/les_fvm/test_emulator.py::test_adjoint_calibration_under_5pct` (`slow`)
- [ ] `tests/les_fvm/test_emulator.py::test_emulated_forward_drop_in`

## Relationships
- Parent (theme epic): #72
- Blocked by: spatial per-cell source control vector (training targets are fields)
- Related: Tier II footprint emulator (same harness pattern)
